### PR TITLE
DOCSP-44808-intellij-installation

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION 
Installation instructions for the IntelliJ Plugin 

## STAGING 
https://deploy-preview-4--docs-mongodb-intellij.netlify.app/install/

## JIRA
https://jira.mongodb.org/browse/DOCSP-44808

### CODE REVIEW

File: source/install.txt [hunk 1]
```diff
@@ -6,10 +6,81 @@ Install the MongoDB for IntelliJ Plugin
...
+To check your current version of IntelliJ IDEA, open the 
+:guilabel:`IntelliJ IDEA` system menu and click :guilabel:`About IntelliJ IDEA`.
+
+Steps 
+-----
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Open the Settings menu
```
Comment (jmd-mongo) [inline source/install.txt]: [comment]

I think all these steps could use a full stop.

File: source/install.txt [hunk 2]
```diff
@@ -6,10 +6,81 @@ Install the MongoDB for IntelliJ Plugin
...
+
+   .. step:: Open the Settings menu 
+
+      Open the :guilabel:`IntelliJ IDEA` system menu and click 
+      :guilabel:`Settings`.
+
+      Alternatively, you can press :kbd:`Cmd` + :kbd:`,` on macOS or 
+      :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`S` on Windows and Linux.
+
+   .. step:: Click on the Plugins tab
```
Comment (jmd-mongo) [inline source/install.txt]: [suggestion]

Consider reframing as: "Show the Plugins tab.". The tab only seems to be revealed once you click **Plugins**.

File: source/install.txt [hunk 3]
```diff
@@ -6,10 +6,81 @@ Install the MongoDB for IntelliJ Plugin
...
+      Alternatively, you can press :kbd:`Cmd` + :kbd:`,` on macOS or 
+      :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`S` on Windows and Linux.
+
+   .. step:: Click on the Plugins tab 
+
+      In the left-hand navigation column, click :guilabel:`Plugins`.
+
+   .. step:: Open the Custom Plugin Repositories modal 
+
+      Next to the :guilabel:`Installed` tab, click the gear icon to open a
```
Comment (jmd-mongo) [inline source/install.txt]: [suggestion]

Consider reordering to start with: "Click the gear icon next to...".

File: source/install.txt [hunk 4]
```diff
@@ -6,10 +6,81 @@ Install the MongoDB for IntelliJ Plugin
...
+         https://plugins.jetbrains.com/plugins/beta/list
+
+   .. step:: Click OK and Restart the IDE 
+
+      After you restart the IDE, open the :guilabel:`Settings` and 
+      navigate to the :guilabel:`Plugins` menu again.
+
+   .. step:: Search for the MongoDB Plugin 
+
+      In the :guilabel:`Markplace` tab of the :guilabel:`Plugins` menu, search
```
Comment (jmd-mongo) [inline source/install.txt]: [suggestion]

Consider noting that folks should only install the official version from MongoDB Inc. There are a few other plugins with very similar names from third party providers.

File: source/install.txt [hunk 5]
```diff
@@ -6,10 +6,81 @@ Install the MongoDB for IntelliJ Plugin
...
+
+      After you restart the IDE, open the :guilabel:`Settings` and 
+      navigate to the :guilabel:`Plugins` menu again.
+
+   .. step:: Search for the MongoDB Plugin 
+
+      In the :guilabel:`Markplace` tab of the :guilabel:`Plugins` menu, search 
+      for ``MongoDB``. 
+
+   .. step:: Click the Install button
```
Comment (jmd-mongo) [inline source/install.txt]: [suggestion]

I think we should also mention to click OK, otherwise the installation may not complete.

**Review by jmd-mongo — COMMENTED** [review]
Thanks for these updates, @ajhuh-mdb! I've left a few comments and suggestions for your consideration.

Thanks,
Joe

File: source/install.txt (hunk 1 above)
Comment (ajhuh-mdb) [reply source/install.txt]: Done!

File: source/install.txt (hunk 2 above)
Comment (ajhuh-mdb) [reply source/install.txt]: Updated!

File: source/install.txt (hunk 3 above)
Comment (ajhuh-mdb) [reply source/install.txt]: Gotcha, that sounds much clearer/smoother! Updated.

File: source/install.txt (hunk 4 above)
Comment (ajhuh-mdb) [reply source/install.txt]: Ah noted! I'll add an `.. important:` admonition there.

File: source/install.txt (hunk 5 above)
Comment (ajhuh-mdb) [reply source/install.txt]: Added!

**Comment by ajhuh-mdb** [comment]
Thanks for the thorough review @jmd-mongo! This should be ready for another look 👍

**Review by jmd-mongo — APPROVED** [review]
Thanks for these additional updates, @ajhuh-mdb! This one LGTM at this point.

Thanks,
Joe

File: source/install.txt [hunk 6]
```diff
@@ -6,10 +6,89 @@ Install the MongoDB for IntelliJ Plugin
...
+      navigate to the :guilabel:`Plugins` menu again.
+
+   .. step:: Search for the MongoDB Plugin. 
+
+      In the :guilabel:`Marketplace` tab of the :guilabel:`Plugins` menu, search 
+      for ``MongoDB``. 
+
+      .. important:: 
+
+         Make sure to download only the official MongoDB plugin from MongoDB Inc
```
Comment (GaurabAryal) [inline source/install.txt:83]: @ajhuh-mdb – does it make sense to include the screenshot here as well? Want to make sure users can identify the official plugin vs not (granted the checkmark should already be a give-away).

**Review by GaurabAryal — APPROVED** [review]
LGTM, thanks!

File: source/install.txt (hunk 6 above)
Comment (ajhuh-mdb) [reply source/install.txt:83]: ah that's a good idea! will add that.

**Pull request merged**